tests/manage/pv_services/test_resource_deletion_during_pvc_pod_deletion_and_io.py

```diff
@@ -447,3 +448,7 @@ def test_disruptive_during_pod_pvc_deletion_and_io(
         # Check ceph status
         ceph_health_check(namespace=config.ENV_DATA['cluster_namespace'])
         log.info("Ceph cluster health is OK")
+
+        log.info("Checking nodes status, bug #1745022")
+        wait_for_nodes_status(node_names=None,status=constants.NODE_READY,timeout=90)
```

Comment:
`node_names=None` and `status=constants.NODE_READY` are already the default values, no need to pass them as such.

```suggestion
        wait_for_nodes_status(timeout=90)
```